msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -720,3 +720,20 @@ You can define any of the following properties:
 * `developerUrl`: The HTTPS URL of the developer's website.
 * `privacyUrl`: The HTTPS URL of the developer's privacy policy.
 * `termsOfUseUrl`: The HTTPS URL of the developer's terms of use.
+
+## defaultBlockUntilAdminAction
```

Comment:
This is available in manifest schema version 1.11. We need to change the version in this document. 
![image](https://user-images.githubusercontent.com/31851992/134318164-b547db95-224f-49f3-ad0c-f9793fd47430.png)

Please check with Surbhi once.